components/BlurryPoster/BlurryPoster.styles.tsx

```diff
@@ -9,7 +9,7 @@ export const blurWrapper = (
 ) => cnb(
   'w-full h-full', {
     'backdrop-blur-md' : addBgBlur,
-    'bg-black-true/40': type === 'hero' && addDarkOverlay && bgColor === 'black',
+    'bg-black-true/60 md:bg-black-true/40': type === 'hero' && addDarkOverlay && bgColor === 'black',
```

Comment:
Since we have a normal weight text link in the masthead now and it gets pretty small on mobile, I'm increasing the dark overlay on story heroes for accessibility contrast.